audioread/rawread.py

```diff
@@ -57,8 +57,10 @@ class RawAudioFile(object):
     """An AIFF, WAV, or Au file that can be read by the Python standard
     library modules ``wave``, ``aifc``, and ``sunau``.
     """
-    def __init__(self, filename):
+    block_samples = None
```

Comment:
Any particular need for this class-level definition? It should be overwritten immediately in the `__init__` call.

Reply:
Ah, it's a habit I got from typescript. I'll nix it